src/app/(auth)/register/page.tsx

```diff
@@ -0,0 +1,63 @@
+"use client";
```

Comment:
move page.tsx into CPage.tsx look how other people do it
Vercel can't deploy if page.tsx is clientside